src/Common/src/Interop/Windows/Crypt32/Interop.certificates_imports.cs

```diff
@@ -0,0 +1,26 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
```

Comment:
If the interop file is being used as a collection of related methods, the associated types should just live in the same file; and the filenames should be Interop.[purpose].cs, not with an _imports/_types suffix.

I'm not really opposed to the _types part, personally (@StephenToub: your thoughts?), but "certificates_imports" to me meant "methods for importing certificates".  All Import.[something].cs files are assumed to be method imports.